x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml

```diff
@@ -383,7 +383,7 @@ setup:
   - is_true: acknowledged
 
   - do:
-      catch: /Downsampling interval \[1h\] must be greater than the the source index interval \[1h\]/
+      catch: /Downsampling interval \[1h\] must be greater than the source index interval \[1h\]/
```

Comment:
Those changes are not relevant to the ILM valiations. I only spotted a typo and fixed it.